README.md

```diff
@@ -100,52 +169,75 @@ and from then on a simple
 
 is enough because the logger has been decorated with the request UUID already and from now on carries this information around.
 
+In other cases however, it might be useful to attach some meta-data to only one message and that can be achieved by using the `metadata` parameter from above:
+
+    logger.trace("user \(userID) logged in",
+                 metadata: ["extra_user_info": [ "favourite_colour": userFaveColour,
+                                                 "auth_method": "password" ]])
+
+The above invocation will log the message alongside meta-data merged from the `logger`'s meta-data and the meta-data provided with the `logger.trace` call.
```

Comment:
Hmmm, in current proposal that's up to the LogHandler impl to decide if it merges or does something else.
But I agree on spelling out "merge" as the most typical prominent case.

Reply:
you are right, I thought that's okay because we can't force them anyway... But if you have a better phrasing I'd probably be okay with that too